### DISCUSSION TITLE
Renaming Motherboard Temprature Sensors?

### DISCUSSION THREAD

**Discussion by SimonZerafa** [Rem0o/FanControl.Releases#968] [Q&A]
Hi,

Using the base app (no plug-ins at the moment) however is there anyway to rename the generic motherboard temperature sensors?

My MSI X570 motherboard has sensors for VRM and other items however the sensors list is fixed with generic labels such as "Temperature #1" etc. There are four detected however they cannot be renamed to match the actual sensors reported buy other apps such as HWinfo.

Is this already a feature I have missed in the FanControl UI or via some other mechanism? 

If not then can this be added as future in a future build? 

Kind Regards

SImon

![Screenshot 2022-07-23 130718](https://user-images.githubusercontent.com/6898637/180604217-35df2235-86a1-4f80-8dfb-e466cd7dfe46.png)

**Comment by Rem0o** [comment]
![image](https://user-images.githubusercontent.com/14118956/180615363-48a8b4a5-e9cf-4770-941b-2308aeb4a7ba.png)

**Comment by verybadsoldier** [comment]
That's great, thank you!
